### ISSUE TITLE
Error with heavy armor's penalties

### ISSUE THREAD

**Issue by wonderfulrabbit** [EndlesNights/dnd4eBeta#134]
This error happens with heavy armor, when there are two penalities, for movement and for skills.

When you put -1 on both fields, only movement decreases. Skill checks get a bonus instead.

**Comment by ahoward-cf** [comment]
This is an inconsistency with how bonuses and penalties are applied, and I
was planning to raise it myself at some point.

@EndlesNights, I am tempted to say that bonus boxes and penalty boxes
should accept either positive or negative numbers, but behind the scenes,
all numbers are made absolute, and then penalty boxes subtract that number,
and bonus boxes add it. Right now, some penalty boxes add the number put
into them, and some subtract it, so if someone puts in -1, it can end up
SUBTRACTING -1, leading to a BONUS of 1.

That or all bonus/penalty boxes are renamed to modifier, and all behave
consistently by adding the inputted number.

**Comment by draconas1** [comment]
Ah bother that will be all the penalties I entered when I did the SRD data entry.  Sorry, my bad for not testing them.  By the time I had finished I was so done with them all :(

Were it greenfield I would suggest the latter because that's nice and logical, but because people are already coping with the existing behaviour and will have arranged their penalties appropriately I think that for this we take the former approach, absolute the number and subtract it.
